### PR TITLE
Add more examples to the test suite

### PR DESCRIPTION
**Pull request by kinnala**
Trying to get these examples to run in Travis.

### CODE REVIEW

**Comment by gdmcbain** [comment]
LGTM but I'm not authorized to approve this on a mobile (which is a reasonable policy). A positive desktop review will have to wait till next week. 

Do you suppose that this technique would reasonably extend to pyamg or pyamgcl?

**Comment by kinnala** [comment]
Yes, I think so. In general, I'd like to add a separate "testing requires" section to setup.py but that has to wait until I'm back at the university so this approach (installing dependencies in .travis.yml) can be used in the meanwhile

**Comment by kinnala** [comment]
Now it also fixes #275 with a temporary solution.

File: skfem/importers/meshio.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 """Import any formats supported by meshio."""
 
 from typing import Tuple
+from collections import OrderedDict
```
Comment (gdmcbain) [inline skfem/importers/meshio.py]: I had to check whether `OrderedDict` was still required or whether a standard [`dict`](https://docs.python.org/dev/library/stdtypes.html#dict) would suffice.  It is till support for 3.6 is abandoned.

> Changed in version 3.7: Dictionary order is guaranteed to be insertion order.

Comment (kinnala) [reply skfem/importers/meshio.py]: Ok, let's have it there for now then. I think 3.6 is supported officially until the beginning of 22?

**Comment by kinnala** [comment]
Is this approved?

**Comment by kinnala** [comment]
I'll postpone merge until I have time to investigate #276 and why it fails on my computer.

**Pull request merged**